src/core/io/writer/h5md/h5md_core.hpp

```diff
@@ -94,8 +95,11 @@ class File
         std::string &scriptname() { return m_scriptname; };
         // Returns the int that describes which data should be written to the dataset.
         int &what() { return m_what; };
+        // Returns the boolean value that describes whether data should be written to the dataset in the order of ids (possibly slower on output for many particles).
+        bool &write_ordered() {return m_write_ordered;} ;
 
     private:
+    	MPI_Comm MPI_H5MD_COMM;
```

Comment:
The variable should be called m_comm or something like that. See https://github.com/espressomd/espresso/wiki/Naming-conventions-and-folder-structure